react/game/src/app/components/Dashboard/index.js

```diff
@@ -1,6 +1,6 @@
 import React, { Fragment } from 'react';
 import { Switch, Route } from 'react-router-dom';
-import routes from '@constants/routes';
+import ROUTES from '@constants/routes';
```

Comment:
```suggestion
import routes from '@constants/routes';
```
Since routes is a object, not a constant.

Reply:
Copy that.